src/main/java/com/mg105/entities/Item.java

```diff
@@ -1,5 +1,4 @@
 package com.mg105.entities;
-
```

Comment:
For completeness sake, we generally avoid random whitespace commits...

(this will not affect the approval, don't go back and change this)